framework/src/main/java/org/apache/drill/test/framework/Utils.java

```diff
@@ -117,6 +118,50 @@
     drillProperties = ImmutableMap.copyOf(properties);
   }
 
+
+ /**
+ * Constructs an iteration of test case definitions from various test data
+ * sources, obtained from the mvn command line option that includes support for regex as well..
```

Comment:
Change to something like "Check if the value passed to the -s parameter (test definition sources) can be expanded as a regex expression and use that to construct the list of test definitions"